index.html

```diff
@@ -65,6 +74,7 @@
 		<img class="desc-image" src="{{ "images/laser.png" | relative }}" />
 		<div class="desc-text">
 			<p>
+				<!-- I would list more than two modules here; probably change the picture to something displaying more than just the image -->
```

Comment:
Yeah, good point. Would something [like this](https://github.com/SquidDev-CC/plethora/blob/minecraft-1.8.9/images/Laser.png) or [this](https://github.com/SquidDev-CC/plethora/blob/minecraft-1.8.9/images/Modules.png) be a better image?

I guess I could also mention how most modules have an in-game use too.